spec/models/user_spec.rb

```diff
@@ -591,13 +591,6 @@
       expect(organizer.editable_fields_of_user(registration.user).to_a).to eq [:name]
     end
 
-    it "allows senior delegates to assign delegate status" do
-      user = FactoryBot.create :user
-      senior_delegate = (FactoryBot.create :senior_delegate_role).user
-      expect(senior_delegate.can_edit_user?(user)).to eq true
-      expect(senior_delegate.editable_fields_of_user(user).to_a).to include(:delegate_status, :region_id, :location)
-    end
-
```

Comment:
@gregorbg This test-case is already there for user-roles update method: https://github.com/thewca/worldcubeassociation.org/blob/04d99b188ad39c00ff317afdb432efd89fe0b2ff/spec/controllers/api/v0/user_roles_controller_spec.rb#L53

I assume both are doing somewhat same thing. Please correct me if I'm wrong.

Reply:
Nah, should be fine! :+1: